pkg/signature/kms/cliplugin/common/options.go

```diff
@@ -27,6 +27,12 @@ type SignOptions struct {
 	MessageOptions MessageOptions `json:"messageOptions"`
 }
 
+// VerifyOptions contains the values for signature.VerifyOption.
+type VerifyOptions struct {
```

Comment:
What is the motivation for defining another structure? Can we just have an `Options` structure with common options?

Reply:
We could, but then the JSON schema would also change, making things a bit more complicated for plugin authors of other languages that would need to parse the json objects directly.

Reply:
OK, I see you're mirroring the existing APIs as well.